src/sentry/runner/commands/run.py

```diff
@@ -427,20 +427,35 @@ def post_process_forwarder(**options):
     type=click.Choice(["earliest", "latest"]),
     help="Force subscriptions to start from a particular offset",
 )
+@click.option(
+    "--use-arroyo-consumer",
+    default=False,
+    is_flag=True,
+    help="Switches to the new arroyo consumer implementation.",
+)
 @log_options()
 @configuration
 def query_subscription_consumer(**options):
-    from sentry.snuba.query_subscription_consumer import QuerySubscriptionConsumer
-
-    subscriber = QuerySubscriptionConsumer(
-        group_id=options["group"],
-        topic=options["topic"],
-        commit_batch_size=options["commit_batch_size"],
-        commit_batch_timeout_ms=options["commit_batch_timeout_ms"],
-        initial_offset_reset=options["initial_offset_reset"],
-        force_offset_reset=options["force_offset_reset"],
+    from sentry.snuba.query_subscription_consumer import (
+        QuerySubscriptionConsumer,
+        get_query_subscription_consumer,
     )
 
+    if not options["use_arroyo_consumer"]:
+        subscriber = QuerySubscriptionConsumer(
+            group_id=options["group"],
+            topic=options["topic"],
+            commit_batch_size=options["commit_batch_size"],
+            commit_batch_timeout_ms=options["commit_batch_timeout_ms"],
+            initial_offset_reset=options["initial_offset_reset"],
+            force_offset_reset=options["force_offset_reset"],
+        )
+    else:
+        subscriber = get_query_subscription_consumer(
```

Comment:
Pass the 2 offset reset options (--auto-offset-reset and --no-strict-offset-reset) into here too from the CLI

Reply:
Where do I use them?

Reply:
Pass to KafkaConsumer. Example of how this: https://github.com/getsentry/sentry/blob/b134ebb0d4cbe3d5a03cd633d4bbb1c29cbd8545/src/sentry/eventstream/kafka/backend.py#L245-L252